source/includes/debt-settlement.md.erb

```diff
@@ -587,31 +586,43 @@ Content-Type: application/json
+| investorRepayments[].investmentId | string | The ID of the of the investment as specified in the original investment creation call                  | required |
+| investorRepayments[].type         | string | The type of the repayment Possible values are: <br>`CAPITAL`<br>`INTEREST`<br>                         | required |
+| investorRepayments[].amount       | ref    | The amount being repaid                                                                                | required |
+| investorRepayments[].tax          | ref    | The amount of tax being withheld from this repayment                                                   | optional |
 ### Response
 | Name                     | Type   | Description                                  |
 | ------------------------ | ------ | -------------------------------------------- |
 | repaymentBatchReference  | string | The unique ID for this repayment request     |
 
+### Possible HTTP Statuses 
+
+| HTTP Status Code | Description                                                                                                                                                                                                                                                                                                | Body                                      | Content-Type     |                 
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------ |------------------| 
+| 200 Ok           | Batch accepted to be instructed. Note that this does not indicate completion or success. Subsequent processing is asynchronous, so the caller should listen to the webhooks detailed above to determine progress.                                                                                          | See above                                 | application/json |                 
+| 400 Bad Request  | The request was malformed. See body for more information                                                                                                                                                                                                                                                   | <a href="/#errors-bad-request">Error</a>  | application/json |        
+| 50x Unauthorized | Unknown errors and timeouts. The caller should not assume that the request has failed. Listen for any subsequent webhooks indicating processing may have started, and ensure any automated retries use the X-CLIENT-REQUEST-ID header and repeat within a maximum of 24 hours to prevent double-processing | <a href="/#errors-unauthorised">Error</a> | application/json |       
```

Comment:
401 is unauthorised.
50x are system and gateway errors

Reply:
Looks like the 401 line got lost!

Reply:
I'm less sure about 401 as it's a given with all our endpoints and the remediation isn't special to this ednpoint specifically whereas dealing with 50x class errors is.